CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
   found in `src`, `href`, `action`, `assetpath` and `style` attributes
   found inside `<template>` elements, when inlining html imports.  Previously,
   this was done by default.  Now requires explicit option.
+- BREAKING: inlinining functions in `import-utils` require an explicit
+ `analyzer` argument.
+- BREAKING: Dropped support for node 4 in latest `polymer-analyzer`:
```

Comment:
That also means that we've dropped support for node v4 here. I'd just copy the lint from analyzer:

* BREAKING: Dropped support for node v4, added support for node v8. See our [node version support policy](https://www.polymer-project.org/2.0/docs/tools/node-support) for details.